testutil/testclient/testblock/client.go

```diff
@@ -32,6 +33,31 @@ func NewLocalnetClient(ctx context.Context, t *testing.T) client.BlockClient {
 	return bClient
 }
 
+// NewAnyTimesCommittedBlocksSequenceBlockClient creates a new mock BlockClient.
+// This mock BlockClient will expect any number of calls to CommittedBlocksSequence,
+// and when that call is made, it returns the given BlocksObservable.
+func NewAnyTimesCommittedBlocksSequenceBlockClient(
```

Comment:
Not actionable, just thinking out loud. We have `OneTimes` and `AnyTimes` in most of these function headers. Thoughts on passing in a `times` variable and defaulting to `AnyTimes` if it's `math.Inf` or `-1`. I don't think this even warrants a TODO but just in case we end up seeing it becomes too verbose in the future.

Reply:
We've discussed this previously, my concern is that I'm not super confident on the direction these testutils will need to develop in and am subsequently reluctant to impose design constraints for scenarios we haven't yet encountered. My main  concern with taking "times" args is that any given test helper may register expectations for multiple methods, in which case we could easily see an explosion of times args across our test helper signatures.

Reply:
Makes complete sense to me!